category.go

```diff
@@ -52,7 +46,7 @@ func (client *Client) GetCategory(id ID) (*Category, error) {
 	}
 	err := client.Query(&q, v, WithName("CategoryGet"))
 	if q.Account.Category.Id == "" {
-		err = fmt.Errorf("Category with ID '%s' not found!", id)
+		err = fmt.Errorf("category with ID '%s' not found", id)
```

Comment:
Starting errors with capital letters and ending with punctuation is discouraged in Go.